### PR TITLE
Added constants for the error messages src/resolver/Mutation

### LINKED ISSUE #1039: Adding constants for the error messages`src/resolver/mutation`
**Is your feature request related to a problem? Please describe.**

This is a child issue to the issue listed here: #1035 
 - The code needs to be changed in `src/resolver/mutation`.
 - constant are values that are already defined for the different values of error message/code/param that are then further used by importing the src/constants folder.

**Describe the solution you'd like**
- we have to define constants for all the hard-coded message/code/param exist in this file. 
-  Refer to the parent issue on getting an insight on how this issue need to be resolved

**Important**
 - Please refer to the parent issues on how to refactor the code correctly.

### PR DESCRIPTION
**Pull request by Manoramsharma**
**What kind of change does this PR introduce?**
Refactoring

**Issue Number:**

Fixes #1039 

**Summary**
https://github.com/PalisadoesFoundation/talawa-api/issues/1039

**Other information**
Changes majorly done in the `src/constants ` file where the constant values were added for the error messages which were passed as the hardcoded strings in different files of `src/resolver/Mutation` folder.
**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Review by noman2002 — CHANGES_REQUESTED** [review]
Fix the failing test.

File: src/constants.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const MEMBER_NOT_FOUND_CODE = "member.notFound";
 export const MEMBER_NOT_FOUND_MESSAGE = "member.notFound";
 export const MEMBER_NOT_FOUND_PARAM = "member";
 
-export const MEMBERSHIP_REQUEST_NOT_FOUND = "Membership Request not found";
+export const MEMBERSHIP_REQUEST_NOT_FOUND = "MembershipRequest already exists";
```
Comment (noman2002) [inline src/constants.ts:32]: Why this text is changed?

**Review by Kevoniat — APPROVED** [review]

File: src/constants.ts (hunk 1 above)
Comment (Manoramsharma) [reply src/constants.ts:32]: I thought by adding extra constant just  for the single message "MembershipRequest already exists" will be affecting the test coverage.

Comment (Manoramsharma) [reply src/constants.ts:32]: Should I create one constant as 
`export const MEMBERSHIP_REQUEST_ALREADY_EXISTS = "MembershipRequest already exists";` then use it in `[src/resolvers/Mutation/sendMembershipRequest.ts]`

Comment (xoldyckk) [reply src/constants.ts:32]: @Manoramsharma Why would name the variable that stores the message `MembershipRequest already exists` as `MEMBERSHIP_REQUEST_NOT_FOUND`? Does it make sense?

Comment (Manoramsharma) [reply src/constants.ts:32]: > @Manoramsharma Why would name the variable that stores the message `MembershipRequest already exists` as `MEMBERSHIP_REQUEST_NOT_FOUND`? Does it make sense?

Yeah, I will be adding a separate variable MEMBERSHIP_REQUEST_ALREADY_EXISTS that stores that value.

**Review by noman2002 — DISMISSED** [review]

**Review by Kevoniat — APPROVED** [review]

**Comment by palisadoes** [comment]
Please fix the conflicting files

**Pull request closed without merging**